saleor/graphql/payment/mutations.py

```diff
@@ -194,6 +256,9 @@ def perform_mutation(cls, _root, info, checkout_id=None, token=None, **data):
             customer_ip_address=get_client_ip(info.context),
             checkout=checkout,
             return_url=data.get("return_url"),
+            store_payment_method=data.get("store_payment_method")
```

Comment:
Not sure if this `OR` is necessary here. `input['store_payment_method']` has default value to `StorePaymentMethod.NONE`, so there is no way to have `None` here (AFAIK)